chain-adapter/src/main/kotlin/com/d3/chainadapter/provider/LastReadBlockProvider.kt

```diff
@@ -0,0 +1,8 @@
+package com.d3.chainadapter.provider
+
+import iroha.protocol.BlockOuterClass
+
+interface LastReadBlockProvider {
+    fun getLastBlock(): Long
```

Comment:
there is some sort of inconsistency in my opinion
we read `Long` but save `Block`